net-dialup/freeradius-client/freeradius-client-1.1.7-r2.ebuild

```diff
@@ -15,7 +15,10 @@ KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ppc ppc64 ~riscv sparc x86"
 
 IUSE="scp shadow static-libs"
 
-DEPEND="!net-dialup/radiusclient-ng"
+DEPEND="
+	!net-dialup/radiusclient-ng
+	virtual/libcrypt
```

Comment:
Needs subslot dep (:=), will fix on merging. Thank you!

https://wiki.gentoo.org/wiki/Project:Toolchain/libcrypt_implementation#Developer_information